lib/kubernetes-deploy/kubectl.rb

```diff
@@ -27,7 +26,6 @@ def run(*args, log_failure: nil, use_context: true, use_namespace: true, output:
       output_is_sensitive = @output_is_sensitive_default if output_is_sensitive.nil?
 
       args = args.unshift("kubectl")
-      args.push("--kubeconfig=#{@kubeconfig}")
```

Comment:
This is the revert (see https://github.com/Shopify/kubernetes-deploy/pull/428/files#diff-f5478b2ac59d4babc30470442f3a8656)